### ISSUE TITLE
About performance

### ISSUE THREAD

**Issue by qianyue76** [shenao-zhang/BARL#1]
I have read your paper and also believe that the solution you proposed is more fundamental than the original reinforcement learning based on the Markov assumption. Theoretically, the expected results should be much better. However, looking at the experimental results, the accuracy is only one to two percentage points higher than other methods. To be honest, this result somewhat weakens the claim that the proposed scheme in the paper is truly "beyond". A little sad :(

**Comment by shenao-zhang** [comment]
Thanks for your interest in our paper! Our performance is reported as the average across three random seeds, so the result is statistically significant. Besides, the GRPO and the Progress method are both very strong baselines. In addition to the LLM experiments, which contain many confounders such as the unknown training data of the base model, please refer to our controlled experiment in synthetic settings in Sec. 6 that clearly demonstrates the benefits of our method. Thanks again for your interest!
